src/command/CommandManager.js

```diff
@@ -83,6 +104,7 @@ define(function (require, exports, module) {
 
     /**
      * Get command id
+     * @private
```

Comment:
The Command class is effectively public as an object is constructed by `register` API and returned. So we need to retain it as public. Modify all API usages below as public too wherever relevant.